qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -317,6 +330,41 @@ def _init_profiles():
         private_profile.setter.init_profile()
 
 
+def _init_site_specific_quirks():
+    if not config.val.content.site_specific_quirks:
+        return
+
+    # default_ua = ("Mozilla/5.0 ({os_info}) "
```

Comment:
Does this comment need to stay here?

Reply:
I'd like to keep it, as it makes it easier to add new user agent strings below by copy-pasting this one and adjusting it in some way.